tests/system/aiplatform/test_model_monitoring.py

```diff
@@ -139,7 +139,7 @@ def temp_endpoint(self, shared_state):
         )
 
         model = aiplatform.Model.upload(
-            display_name=MODEL_NAME,
+            display_name=self._make_display_name(key=MODEL_NAME),
             artifact_uri=CHURN_MODEL_PATH,
             serving_container_image_uri=IMAGE,
         )
```

Comment:
When the model is being deployed, on the line `endpoint = model.deploy(machine_type="n1-standard-2")`, I think endpoint is (if unspecified) set to model display name+'_endpoint'. Want to make sure with you that this is the behavior you're looking for. Works for me

Reply:
yep that's the intended behavior; no need to specify the endpoint name here because we're passing the endpoint object itself to the MDM job creation call